cibuildwheel/util.py

```diff
@@ -562,6 +565,46 @@ def virtualenv(
     return env
 
 
+@overload
+def find_abi3_wheel(wheels: Sequence[Path], identifier: str) -> Optional[Path]:
+    ...
+
+
+@overload
+def find_abi3_wheel(wheels: Sequence[PurePath], identifier: str) -> Optional[PurePath]:
+    ...
+
+
+def find_abi3_wheel(
```

Comment:
Minor suggestion: `find_compatible_abi3_wheel(identifier, wheels)` might be clearer?

Reply:
will do

Reply:
done